vendor/tests/test_processor.py

```diff
@@ -818,7 +818,7 @@ def test_create_subscription_success(self):
     def test_subscription_update_payment(self):
         self.form_data['credit_card_form']['card_number'] = choice(self.VALID_CARD_NUMBERS)
         subscription_list = self.processor.get_list_of_subscriptions()
-        if not subscription_list:
+        if not len(subscription_list):
```

Comment:
I _think_ an empty list will resolve to false in a python `if` statement

Reply:
I added the len functions as I saw a warning that it would be safer to add the len to check if an list was empty with len